libraries/SITL/SIM_Sailboat.cpp

```diff
@@ -97,13 +100,19 @@ float Sailboat::get_turn_circle(float steering) const
 // return yaw rate in deg/sec given a steering input (in the range -1 to +1) and speed in m/s
 float Sailboat::get_yaw_rate(float steering, float speed) const
 {
-    if (is_zero(steering) || is_zero(speed)) {
-        return 0;
+    float rate = 0.0f;
+    if (is_zero(steering) || (!skid_steering && is_zero(speed))) {
+        return rate;
+    }
```

Comment:
Thanks for this.  Just a nitpick on style, we normally put the else on the same line as the preceding curly bracket

Reply:
Fixed